### DISCUSSION TITLE
run eslint and get error: There was a problem loading formatter: ...\node_modules\eslint\lib\cli-engine\formatters\stylish

### DISCUSSION THREAD

**Discussion by MH4GF** [eslint/eslint#17269] [Q&A] [answered]
Hi there! Thanks for maintaining eslint 😄 

## Environment
Node version: v18.12.1
npm version: 8.19.2
Local ESLint version: v8.42.0 (Currently used)
Global ESLint version: Not found
Operating System: Mac OS

## What parser are you using?

TypeScript Parser (@typescript-eslint/parser)

## What did you do?

When I run eslint, I get the following error:

```
yarn run eslint src
yarn run v1.22.19
$ /Users/MH4GF/my-package/node_modules/.bin/eslint src
There was a problem loading formatter: /Users/MH4GF/my-package/node_modules/eslint/lib/cli-engine/formatters/stylish
Error: require() of ES Module /Users/MH4GF/my-package/node_modules/strip-ansi/index.js from /Users/MH4GF/my-package/node_modules/eslint/lib/cli-engine/formatters/stylish.js not supported.
Instead change the require of index.js in /Users/MH4GF/my-package/node_modules/eslint/lib/cli-engine/formatters/stylish.js to a dynamic import() which is available in all CommonJS modules.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## Link to Minimal Reproducible Example

https://github.com/MH4GF/dependency-cruiser-report-action/pull/244
https://github.com/MH4GF/dependency-cruiser-report-action/actions/runs/5222658817/jobs/9428439091?pr=244

It appears to be caused by an update to v13.0.0 of dependency-cruiser, which this repository depends on.

https://github.com/eslint/eslint/discussions/17215#discussioncomment-5985393 The situation is the same as what is pointed out in this comment, and dependency-cruiser also uses `glob@10.2.6` from this version. 

## Additional comments

related: https://github.com/eslint/eslint/discussions/17215

**Comment by Rec0iL99** [comment, marked as answer]
Hi @MH4GF, I could reproduce the error, and this is caused by `dependency-cruiser`. I confirmed by removing `dependency-cruiser` to see if ESLint works and it did.

But this does not look like an issue with ESLint since we don't have a direct dependency on `dependency-cruiser`. I feel opening an issue there should help you out here.

**Reply by MH4GF** [reply]
Thank you for response! but my tool is based on dependency-cruiser and I could not get rid of it...
Finally, I was able to solve the problem by change from yarn to pnpm and not hoisting.
